src/main/scala/com/fulcrumgenomics/alignment/Alignment.scala

```diff
@@ -91,33 +91,71 @@ object Cigar {
 }
 
 /** A data class for holding statistics about a [[Cigar]]. */
-private[alignment] case class CigarStats(lengthOnQuery: Int, lengthOnTarget: Int, alignedBases: Int, clippedBases: Int)
+private[alignment] case class CigarStats(lengthOnQuery: Int,
+                                         lengthOnTarget: Int,
+                                         alignedBases: Int,
+                                         clippedBases: Int,
+                                         softClippedFivePrimeBases: Int,
+                                         softClippedThreePrimeBases: Int)
```

Comment:
The way these are calculated, they are not 5' and 3', they are "leading" and "trailing" or "starting" and "ending".